addons/stock/report/report_stock_forecast.xml

```diff
@@ -55,7 +55,7 @@
         <field name="view_type">form</field>
         <field name="view_mode">pivot,graph</field>
         <field name="search_view_id" ref="view_stock_level_forecast_filter"/>
-        <field name="context">{'search_default_pivot_by':1, 'search_default_graph_by':1, 'search_default_product_id': active_id}</field>
+        <field name="context">{'search_default_pivot_by':1, 'search_default_graph_by':1, 'search_default_product_id': active_id, group_by': ['company_id']}</field>
```

Comment:
@simongoffin I tested this commit and I think it is wrong. A quote is missing before groupby